src/language-css/printer-postcss.js

```diff
@@ -672,22 +683,44 @@ function genericPrint(path, options, print) {
             concat([
               softline,
               join(
-                concat([
-                  ",",
-                  declNode && isSCSSMapNode(declNode) ? hardline : line
-                ]),
-                path.map(print, "groups")
+                concat([",", line]),
+                path.map(childPath => {
+                  const node = childPath.getValue();
+                  const printed = print(childPath);
+
+                  // Key/Value pair in open paren already indented
+                  if (
+                    isKeyValuePairNode(node) &&
+                    node.type === "value-comma_group" &&
+                    node.groups &&
+                    node.groups[2] &&
+                    node.groups[2].type === "value-paren_group"
+                  ) {
+                    printed.contents.contents.parts[1] = group(
+                      printed.contents.contents.parts[1]
```

Comment:
This feels like a hack?

Reply:
@lydell we have `property:<line>value` and we need use `group` on then because we break all group.
i.e.
Without hack:
```scss
$map: (
  key: 
    (
      1,
      2 
    )
);
```

Without hack:
```scss
$map: (
  key: (
      1,
      2 
  )
);
```

In theory we can add check `isSCSSMapNodeItem` in `value-colon` and don't add `line` when we in SCSS map.